pkg/cmd/experimental/syncgroups/config/types.go

```diff
@@ -0,0 +1,148 @@
+package config
+
+import (
+	ldaputil "github.com/openshift/origin/pkg/auth/authenticator/password/ldappassword/util"
+	"github.com/openshift/origin/pkg/client"
+	"k8s.io/kubernetes/pkg/util"
+)
+
+// LDAPSyncCommonConfig contains all of the config that is common to all LDAP schemas
+type LDAPSyncCommonConfig struct {
+	// ServerConfig holds the configuration options for connecting with the LDAP server
+	ServerConfig ldaputil.LDAPServerConfig
+
+	// ProvisionGroups determines whether or not the GroupSyncer will create new OpenShift Groups if
+	// they do not yet exist or ignore those sync requests that do not map to existing OpenShift Groups
+	ProvisionGroups bool
```

Comment:
I see this is also redundant with a whitelist. That's a better way of getting the same result.